nav2_smac_planner/src/smac_planner_2d.cpp

```diff
@@ -250,28 +250,15 @@ nav_msgs::msg::Path SmacPlanner2D::createPlan(
   // Compute plan
   Node2D::CoordinateVector path;
   int num_iterations = 0;
-  std::string error;
```

Comment:
A note in all 3 planners as a comment that the exceptions thrown by the planners will be captured by the planner servers and states returned to the action requester